test/integration/acl-tls.js

```diff
@@ -161,20 +161,26 @@ describe('ACL HTTP', function () {
       })
     })
     describe('with defaultForNew in parent path', function () {
-      before(function () {
+      before(() => {
         rm('/acl-tls/write-acl/empty-acl/another-empty-folder/test-file.acl')
         rm('/acl-tls/write-acl/empty-acl/test-folder/test-file')
         rm('/acl-tls/write-acl/empty-acl/test-file')
         rm('/acl-tls/write-acl/test-file')
         rm('/acl-tls/write-acl/test-file.acl')
+        rm('/acl-tls/write-acl/empty-acl/test-folder2/')
```

Comment:
You've got this in the `before` and `after`. Not really sure what the intention is here.